src/registration/LabelledTextInput.tsx

```diff
@@ -0,0 +1,26 @@
+import { useUniqueId } from '@terra-ui-packages/components';
+import React, { CSSProperties } from 'react';
+import { TextInput } from 'src/components/input';
+import { FormLabel } from 'src/libs/forms';
+
+interface LabelledTextInputProps {
+  disabled?: boolean;
+  required?: boolean;
+  value: string;
+  onChange: (value: string) => void;
+  labelStyle?: CSSProperties;
+  inputStyle?: CSSProperties;
+  label: string;
+}
+export const LabelledTextInput = (props: LabelledTextInputProps) => {
```

Comment:
For consistency/convention.

```suggestion
export const LabelledTextInput = (props: LabelledTextInputProps): ReactNode => {
```